Modules/IO/Mesh/include/README.md

```diff
@@ -0,0 +1,3 @@
+This module is present for backwards compatibility and should not be used -
```

Comment:
Why deprecate it? Shouldn't we leave it to have the same purpose as `ITKImageIO` and `ITKTransformIO` meta-groups.

Reply:
For that purpose we have `ITKMeshIO`.

We will be able to keep things simple and add additional formats / modules in the future without it.